databend-client/src/main/java/com/databend/client/data/Int8Handler.java

```diff
@@ -465,6 +465,9 @@ private Float parseNullableValue(Object value) {
         if (value == null || value.equals("NULL")) {
             return null;
         }
+        if (value.equals("NaN") || value.equals("nan")) {
```

Comment:
Int8 do not need to handle Nan.

what need modify is Float32 and Float64

Reply:
> Int8 do not need to handle Nan.
> 
> what need modify is Float32 and Float64

@youngsofun Of course I modify the Float32 and Float64.
![image](https://github.com/datafuselabs/databend-jdbc/assets/7600925/80399d6f-70e3-4ea3-a7f0-c75a9a21f2a5)

Reply:
Sorry, I saw you are modifying Int8Hnalder.java.
the file name is strange.